<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="main_box">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>321</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Исходные данные</string>
    </property>
    <widget class="QLineEdit" name="side_A">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Сторона A</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="side_B">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Сторона B</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="angle_between_sides">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Угол между</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="angle_degree">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>градусы</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="angle_rad">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>145</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>радианы</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="result_box">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>311</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Результат</string>
    </property>
    <widget class="QLabel" name="result_third_side">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="calculate_third_side">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>281</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Рассчитать третью сторону</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
